src/test/daml/Daml/Finance/Settlement/Test/Transfer.daml

```diff
@@ -316,7 +326,84 @@ run4 = script do
 
   pure ()
 
+-- Transfer of commercial bank money from Alice to Charlie via Bob (acting as an intermediary/passthrough).
+-- +---------------------------------------------------+
+-- | Accounts                                          |
+-- +---------------------------+-----------------------+
+-- | structure:                | used for:             |
+-- +---------------------------+-----------------------+
+-- |           Bank            |                       |
+-- |      /     |    \         | commercial bank money |
+-- | Sender -> CCP -> Receiver |                       |
+-- +---------------------------+-----------------------+
+run5 : Script ()
+run5 = script do
+  TestParties{..} <- setupParties
+
+  -- Account and holding factory
+  let fp = [("FactoryProvider", singleton publicParty)]
+  accountFactoryCid <- toInterfaceContractId <$> Account.createFactory provider fp
+  holdingFactoryCid <- toInterfaceContractId <$> submitMulti [publicParty] [] do
```

Comment:
I would avoid using `publicParty` as an `actAs`, given that it never occurs in practice. We can use `provider` or `bank` (see my comment above)

Reply:
agree